nexus-repository-helm-it/src/test/java/org/sonatype/nexus/plugins/helm/internal/HelmProxyIT.java

```diff
@@ -65,61 +68,45 @@ public void tearDown() throws Exception {
 
   @Test
   public void fetchTgzPackageFile() throws Exception {
-    assertSuccessResponseMatches(client.fetch(MONGO_PKG_FILE_NAME_600_TGZ, CONTENT_TYPE_TGZ), MONGO_PKG_FILE_NAME_600_TGZ);
-    final Asset asset = findAsset(repository, MONGO_PKG_FILE_NAME_600_TGZ);
-    Assert.assertThat(asset.name(), is(equalTo(MONGO_PKG_FILE_NAME_600_TGZ)));
-    Assert.assertThat(asset.contentType(), is(equalTo(CONTENT_TYPE_TGZ)));
-    Assert.assertThat(asset.format(), is(equalTo(HELM_FORMAT_NAME)));
+    checkAsset(MONGO_PKG_FILE_NAME_600_TGZ, CONTENT_TYPE_TGZ);
   }
 
   @Test
   public void fetchMetaData() throws Exception {
-    assertSuccessResponseMatches(client.fetch(YAML_FILE_NAME, CONTENT_TYPE_YAML), YAML_FILE_NAME);
-    final Asset asset = findAsset(repository, YAML_FILE_NAME);
-    Assert.assertThat(asset.contentType(), is(equalTo(CONTENT_TYPE_YAML)));
-    Assert.assertThat(asset.format(), is(equalTo(HELM_FORMAT_NAME)));
+    checkAsset(YAML_FILE_NAME, CONTENT_TYPE_YAML);
   }
 
   @Test
   public void checkComponentCreated() throws Exception {
-    final Component nullComponent = findComponent(repository, MONGO_PKG_NAME);
-    Assert.assertThat(nullComponent, is(nullValue()));
+    Assert.assertFalse(componentAssetTestHelper.componentExists(repository, MONGO_PKG_NAME, MONGO_PKG_VERSION_600));
     client.fetch(MONGO_PKG_FILE_NAME_600_TGZ, CONTENT_TYPE_TGZ);
-
-    final Component component = findComponent(repository, MONGO_PKG_NAME);
-    Assert.assertThat(component.name(), is(equalTo(MONGO_PKG_NAME)));
-    Assert.assertThat(component.format(), is(equalTo(HELM_FORMAT_NAME)));
-    Assert.assertThat(component.group(), is(nullValue()));
-    Assert.assertThat(component.version(), is(equalTo(MONGO_PKG_VERSION_600)));
+    Assert.assertTrue(componentAssetTestHelper.componentExists(repository, MONGO_PKG_NAME, MONGO_PKG_VERSION_600));
```

Comment:
static import and remove unused imports please

Reply:
fixed 8402364654feb6ef462e10fada1794abf0a580c6